watches/commands/base.py

```diff
@@ -46,6 +51,142 @@ def getResponseContentType(self):
         """Response MIME type. By default we assume JSON, make sure to override if needed."""
         return self.JSON_APPLICATION
 
+    def transformData(self, data):
+        """
+        Data can be transformed before sending to client.
+        Currently, the only transformation type implemented is 'nested'.
+        :param data:
+        :return:
+        """
+        transform = self.options['--transform']
+        if transform and transform is not None and transform == "nested":
```

Comment:
@richm I updated the code.

Unfortunately, after several attempts I gave up on implementing regular unit test for catching exceptions when unsupported `transform` option is used. _Python is still not my best friend._